src/main/java/com/novauc/Controller.java

```diff
@@ -0,0 +1,86 @@
+package com.novauc;
+
+import org.springframework.boot.autoconfigure.web.ErrorAttributes;
+import org.springframework.core.Ordered;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.servlet.http.HttpSession;
+import java.util.ArrayList;
+
+/**
+ * Created by dangelojoyce on 3/13/17.
+ */
+
+
+public abstract class Controller extends Object implements ErrorAttributes, Ordered
+{
+    private ErrorAttributes errorAttributes;
+
```

Comment:
Controller is a Spring "keyword".  So you can't name your controller class just Controller and annotate it with @Controller, which I'm guessing is why you made it abstract and extend Object (which happens implicitly) etc.  So rename your class something different and then annotate it with @Controller. That should get you back on track.